lib/adm.js

```diff
@@ -2337,27 +2337,49 @@ maAdm.prototype.dumpConfigCoal = function(options, callback) {
             authcache: 1,
             webapi: 1,
             loadbalancer: 1,
-            ops: 1
+            ops: 1,
+            rebalancer: 1,
+            picker: 1,
+            'buckets-postgres': 2,
+            'buckets-mdplacement': 1,
+            'buckets-mdapi': 1,
+            'buckets-api': 1
         },
         callback
     );
 };
 
+/*
+ * Generate a reasonable "lab"-sized Manta deployment configuration. "lab"-sized
+ * here means something bigger (i.e. more RAM) than COAL, but still constrained.
+ *
+ * Goals of the numbers here:
+ * - 3 for nameservice and postgres clusters to test HA
+ * - 2 for moray to have more than one to test potential issues? This is
+ *   a hand-waving argument.
+ * - 1 for other services to both support easier developer debugging (just
+ *   one inst log to go looking at) and to avoid undue usage of limited RAM
+ */
 maAdm.prototype.dumpConfigLab = function(options, callback) {
     return this.dumpConfigCommon(
         options,
         {
             shards: ['1', '2'],
             nameservice: 3,
             postgres: 3,
-            moray: 3,
+            moray: 2,
             'electric-moray': 1,
             storage: 3,
-            authcache: 2,
-            webapi: 2,
-            loadbalancer: 2,
-            madtom: 1,
-            ops: 1
+            authcache: 1,
+            webapi: 1,
+            loadbalancer: 1,
+            ops: 1,
+            rebalancer: 1,
+            picker: 1,
+            'buckets-postgres': 3,
```

Comment:
does this mean 3 for each of the shards (shards: ['1', '2'] above) or... how do these 3 get allocated?

Reply:
nm, answered elsewhere.